docs/start-building/index.txt

```diff
@@ -53,9 +55,9 @@ If you would like to see something added to this page, please `get in touch`_.
 .. _DBAL: https://crate.io/docs/clients/dbal/en/latest/
 .. _DBD::Crate: https://github.com/mamod/DBD-Crate
 .. _get in touch: https://crate.io/contact/
-.. _go-crate: https://github.com/herenow/go-crate
+.. _pgx: https://github.com/jackc/pgx
+.. _asyncpg: https://github.com/MagicStack/asyncpg
```

Comment:
minor nit: please order the links alphabetically

Reply:
For what purpose?

Reply:
it's a style guide item. makes the list easier to scan when editing (esp when it gets long). it also makes changes deterministic

Reply:
Pushed a fixup, wasn't sure how the sort should be done in regards to lower vs. upper case. Now all upper case items are sorted before lower case.

Reply:
I should have specified: case insensitive. I added a fixup of my own as an apology haha. will add a note to the style guide

Reply:
Okay, can you approve then?